packages/desktop-client/src/components/manager/Import.js

```diff
@@ -30,6 +30,9 @@ function Import({ modalProps, actions }) {
       case 'actual':
         actions.pushModal('import-actual');
         break;
+      case 'wallet':
+        actions.pushModal('import-wallet');
```

Comment:
Firstly, thank you for your interest in contributing to Actual! 

Can we rename to something more specific like `import-budgetbakers-wallet`? I feel like wallet is too generic as there might be other finance apps that could also be called wallet. The same applies for other parts of this PR as well.